src/Sparrow/Json/BlittableJsonReaderObject.cs

```diff
@@ -230,10 +232,19 @@ public bool TryGet<T>(StringSegment name, out T obj)
                 obj = default(T);
                 return false;
             }
-            ConvertType(result, out obj);
+            try
+            {
+                ConvertType(result, out obj);
+            }
+            catch
```

Comment:
Not a good idea, we have a value, just different type, and will mask a lot of errors, I fear